redaxo/src/core/lib/util/finder.php

```diff
@@ -237,8 +237,7 @@ public function getIterator()
             }
 
             if ($this->ignoreSystemStuff) {
-                static $systemStuff = ['.DS_Store', 'Thumbs.db', 'desktop.ini', '.svn', '_svn', 'CVS', '_darcs', '.arch-params', '.monotone', '.bzr', '.git', '.hg'];
-                foreach ($systemStuff as $systemStuffFile) {
+                foreach (['.DS_Store', 'Thumbs.db', 'desktop.ini', '.svn', '_svn', 'CVS', '_darcs', '.arch-params', '.monotone', '.bzr', '.git', '.hg'] as $systemStuffFile) {
```

Comment:
Ich vermute, dass auch das performanter ist. Denke, das kann PHP besser optimieren, ein statisches Array, als eine statische Variable, die vorgehalten werden muss und sich sonstwo in der Methode geändert haben könnte (bei vorherigen Aufrufen).

Reply:
Static sollten wir nur verwenden wenn wir wirklich state speichern wollen.

Ich denke wir/ich haben das früher genutzt im glauben man würde sich was sparen - was aus heutiger sicht quatsch ist

Reply:
Ich glaube, viele Stellen sind von mir. Ich hielt das immer für sinnvoll.